firefox_decrypt.py

```diff
@@ -430,6 +431,14 @@ def output_line(line):
         LOG.info("Decrypting credentials")
         to_export = {}
 
+        if format == 'csv':
+            csv_writer = csv.DictWriter(
+                sys.stdout, fieldnames={'host', 'user', 'password'},
+                delimiter=';', quotechar='|', quoting=csv.QUOTE_ALL
```

Comment:
I have a bit of a problem with this line. CSV is not a standard format and using `;` together with `|` is definitely not standard.